lib/earmark/renderers/html_renderer.ex

```diff
@@ -179,25 +187,40 @@ defmodule Earmark.HtmlRenderer do
   # And here are the inline renderers #
   #####################################
 
-  def br,                  do: "<br/>"
-  def codespan(text),      do: ~s[<code class="inline">#{text}</code>]
-  def em(text),            do: "<em>#{text}</em>"
-  def strong(text),        do: "<strong>#{text}</strong>"
-  def strikethrough(text), do: "<del>#{text}</del>"
+  defp render_block(%Inline.Br{}, _), do: "<br/>"
```

Comment:
Rewriting the `HtmlRenderer` is not what I'd favor, seems we got 10% performance loss, not sure if it is because of the modifications in the `HtmlRenderer` or because of the connaissance in inline rendering.

Reply:
That does not mean that the new code is not better, just that the scope of this issue/PR shall not touch it.